checks.d/openstack.py

```diff
@@ -512,3 +650,57 @@ def check(self, instance):
         except IncompleteConfig:
             self.warning("Configuration Incomplete! Check your openstack.yaml file")
             return
+
+
+    #### Local Info accessors
+    def get_local_hypervisor(self):
+        # Look up hypervisors available filtered by my hostname
+        host = self.get_my_hostname()
+        hyp = self.get_all_hypervisor_ids(filter_by_host=host)
+        return hyp[0]
+
+    def get_local_project(self):
```

Comment:
Agreed, it is poorly named. What it means to do is "get the project that this check is scoped to", which it knows from reading the configuration. Will update it to `get_scoped_project` if that sounds closer to reality